auth-web/src/components/auth/IncorpOrRegisterInfo.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <v-container id="incorporate-info-container">
+    <v-row>
+      <!-- Info Column -->
+      <v-col cols="12" md="6">
+        <h2>Incorporate or Register</h2>
+        <v-list-item class="list-item">
+          <v-icon size="6" class="list-item-bullet">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              If you have an approved Name Request (NR number), or you want a <NumberedCompanyTooltip />,
+              you can start your Incorporation Application.
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <v-list-item class="list-item" v-for="(item, index) in bulletPoints" :key="index">
+          <v-icon size="6" class="list-item-bullet">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Panel Btns -->
+        <!-- Authenticated -->
+        <div v-if="userProfile">
```

Comment:
Not mandatory to change, but if you just need an element for a v-if or v-show, use a "template". (Divs are block elements and carry style and an additional layer of depth in the DOM. Templates aren't/don't.)

Reply:
Shouldn't this block have a class, like the one below?

Reply:
oo interesting :) Yea, i will try this out, thanks!

Reply:
Updated in next commit!